httpbin/handlers.go

```diff
@@ -740,19 +737,33 @@ func handleBytes(w http.ResponseWriter, r *http.Request, streaming bool) {
 	rng := rand.New(src)
 
 	w.Header().Set("Content-Type", "application/octet-stream")
+	w.Header().Set("Content-Length", strconv.FormatInt(numBytes, 10))
```

Comment:
Setting the content type prevents us from streaming the body back to the client for the /stream-bytes/ endpoint, because it prevents the stdlib from automatically adding the `Transfer-Encoding: chunked` header.

The easiest way I know to quickly verify this, since I could not figure out how to actually unit test it (see the comment in the deleted test code below, which should also be restored), is to use telnet:

```
telnet localhost 8080
```

Type this in and hit enter a couple of times:

```
GET /stream-bytes/1024?chunk_size=10 HTTP/1.1
Host: foo
```

You should see a `Transfer-Encoding: chunked` header and a bunch of 10 byte chunks instead of a single 1024 byte chunk.

Reply:
Oh I see a misunderstanding on my part here. 
For me 'stream-bytes' is about "streaming bytes" (since there is "application/octet-stream") and not about 'Transfer-Encoding: chunked'. The chunk part is to control the flushing not the 'chunked' behavior as defined in https://tools.ietf.org/html/rfc7230#section-4.1 (aka add a size in hex and carriage returns between each chunks which is what stdlib does hence your 10 byte chunks).

So I removed 'Transfer-Encoding: chunked' because it's usually not really compatible with 'application/octet-stream' (I think it's open to interpretation here since httpbin does it).

 may be there should be 2 different endpoints.